website/docs/reference/login-history.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Login history
+---
+
+:::info
+
+Login history is an enterprise feature available from Unleash 4.22 onwards.
+
+:::
+
+Login history lets you track login events in your Unleash instance, and whether the attempts were successful in logging in or not. 
+
+![Login history table](/img/login-history-table.png)
+
+For each login event, it lists:
+
+ - **Created**: When it happened
+ - **Username**: The username that was used
+ - **Authentication**: The authentication type that was used
+ - **IP address**: The IP address that made the attempt
+ - **Success**: Whether the attempt was successful or not
+ - **Failure reason**: If the attempt was not successful, the reason why
+
+You can see the failure reason by hovering over the "False" badge in the "Success" column.
+
+![Login history table failure reason](/img/login-history-table-fail.png)
+
+Use login history to:
+
+- Audit login events in your Unleash instance
+- Identify failed login attempts and investigate the cause
+- Debug misconfigured authentication providers
+
+## Retention
+
+Events in the login history are retained for 336 hours (14 days) by default. This setting can be found under the key `login_history_retention` in the `settings` table.
```

Comment:
It's the internal `settings` database table. Not sure if I should mention it here 🤔 
Currently we have no way of setting this through the UI because we don't have a "general settings" page.

Reply:
AFAIK currently there's no way of configuring this setting other than directly setting the value in this database table. Maybe @chriswk can chime in with some more valuable input here.

Reply:
Got it! In that case, I'd suggest we remove how it's stored at the moment. How's this? 

```suggestion
Events in the login history are retained for 336 hours (14 days).
```